test/runtests.jl

```diff
@@ -86,3 +90,67 @@ end
     @test logdensity(Dirac(0.3), 0.3) == 0.0
     @test logdensity(Dirac(0.3), 0.4) == -Inf
 end
+
+import MeasureTheory.:⋅
+function ⋅(μ::Normal, kernel) 
```

Comment:
We should have this in the package, not just tests. But we'll need to narrow down the argument types a little bit for it to work